<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MSalman" ignored="0" name="LoginTest" tests="5" failures="3" timestamp="12 Jul 2019 11:51:34 GMT" time="41.814" errors="0">
  <testcase name="TC_Negative_Verify_theError_messageDisplayed_whenUser_EnterInvalid_email" time="14.072" classname="tests.ReactResendConfirmationEmailTest"/>
  <testcase name="TC_Negative_Verify_theError_messageDisplayed_whenEmail_IsEmpty" time="5.409" classname="tests.ReactResendConfirmationEmailTest"/>
  <testcase name="TC_Negative_Verify_theError_messageDisplayed_whenUser_enterUnregistered_email" time="5.614" classname="tests.ReactResendConfirmationEmailTest">
    <failure type="java.lang.AssertionError" message="expected [Success: An email with password reset instructions has been sent to email@domain.com, if it exists on our system] but found [A server error happened. We are working to fix it.]">
      <![CDATA[java.lang.AssertionError: expected [Success: An email with password reset instructions has been sent to email@domain.com, if it exists on our system] but found [A server error happened. We are working to fix it.]
at pages.ReactResendConfirmationEmail.verifySuccessEmail(ReactResendConfirmationEmail.java:57)
at tests.ReactResendConfirmationEmailTest.TC_Negative_Verify_theError_messageDisplayed_whenUser_enterUnregistered_email(ReactResendConfirmationEmailTest.java:110)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_Negative_Verify_theError_messageDisplayed_whenUser_enterUnregistered_email -->
  <testcase name="TC_Negative_Verify_theError_messageDisplayed_whenUser_enterregistered_emailand_ClickonButton" time="5.635" classname="tests.ReactResendConfirmationEmailTest">
    <failure type="java.lang.AssertionError" message="expected [Success: An email with password reset instructions has been sent to test@mailinator.com, if it exists on our system] but found [A server error happened. We are working to fix it.]">
      <![CDATA[java.lang.AssertionError: expected [Success: An email with password reset instructions has been sent to test@mailinator.com, if it exists on our system] but found [A server error happened. We are working to fix it.]
at pages.ReactResendConfirmationEmail.verifySuccessEmail(ReactResendConfirmationEmail.java:57)
at tests.ReactResendConfirmationEmailTest.TC_Negative_Verify_theError_messageDisplayed_whenUser_enterregistered_emailand_ClickonButton(ReactResendConfirmationEmailTest.java:142)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_Negative_Verify_theError_messageDisplayed_whenUser_enterregistered_emailand_ClickonButton -->
  <testcase name="TC_Negative_Verify_theError_messageDisplayed_whenUser_shouldbe_ableToSend_PasswordLink" time="5.458" classname="tests.ReactResendConfirmationEmailTest">
    <failure type="java.lang.AssertionError" message="expected [Success: An email with password reset instructions has been sent to test@mailinator.com, if it exists on our system] but found [A server error happened. We are working to fix it.]">
      <![CDATA[java.lang.AssertionError: expected [Success: An email with password reset instructions has been sent to test@mailinator.com, if it exists on our system] but found [A server error happened. We are working to fix it.]
at pages.ReactResendConfirmationEmail.verifySuccessEmail(ReactResendConfirmationEmail.java:57)
at tests.ReactResendConfirmationEmailTest.TC_Negative_Verify_theError_messageDisplayed_whenUser_shouldbe_ableToSend_PasswordLink(ReactResendConfirmationEmailTest.java:174)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TC_Negative_Verify_theError_messageDisplayed_whenUser_shouldbe_ableToSend_PasswordLink -->
</testsuite> <!-- LoginTest -->
